lib/config.js

```diff
@@ -59,49 +59,78 @@ function isFilePath(filePath) {
 }
 
 /**
- * Load and parse a JSON config object from a file.
+ * Check if item is an javascript object
+ * @param {*} item object to check for
+ * @returns {boolean} True if its an object
+ * @private
+ */
+function isObject(item) {
```

Comment:
I agree. I just wanted to point out if you preferred that. The trade off for saving a dependency is us maintaining this code which looks perfectly okay for this use case :)